enarx-keep-sev-shim/src/main.rs

```diff
@@ -52,11 +53,18 @@ macro_rules! entry_point {
 
 entry_point!(shim_main);
 
+/// FIXME: will be replaced by a dynamically allocated stack with safe guards
+pub static mut LEVEL_0_STACK: [Page; 5] = [Page::zeroed(); 5];
```

Comment:
Should `STACK_NUM_PAGES` be used here?

Reply:
No, because this is the main stack and it will be changed in later patches anyway.